### PR TITLE
Add more Insignias

### PR DESCRIPTION
**Pull request by Lairdd1989**
**When merged this pull request will:**
Adds following insignia

-Brand Of Sacrifice
-Bridge Four
-I Am Unstoppable
-Judge
-Natural Twenty
-Never Forget
-Raptor Squad

https://imgur.com/a/Ju6E4mm

Thanks to Mick and Charley for pointers and tips

### CODE REVIEW

File: addons/insignias/CfgUnitInsignia.hpp [hunk 1]
```diff
@@ -249,7 +261,13 @@ class CfgUnitInsignia {
         displayName = "I'm A Lead Farmer (Arid)";
         texture = QPATHTOF(data\iamaleadfarmer_arid.paa);
     };
-    class CLASS(Insignia_IComeILeave) {
+		class CLASS(Insignia_iamunstoppable) {
```
Comment (TyroneMF) [inline addons/insignias/CfgUnitInsignia.hpp:264]: Indentation & Tabs.

File: addons/insignias/CfgUnitInsignia.hpp [hunk 2]
```diff
@@ -273,6 +291,12 @@ class CfgUnitInsignia {
         displayName = "Jedi Order Shield";
         texture = QPATHTOF(data\jediordershield.paa);
     };
+		class CLASS(Insignia_judge) {
```
Comment (TyroneMF) [inline addons/insignias/CfgUnitInsignia.hpp:294]: Again.

File: addons/insignias/CfgUnitInsignia.hpp [hunk 3]
```diff
@@ -369,6 +393,18 @@ class CfgUnitInsignia {
         displayName = "Murica (Multicam)";
         texture = QPATHTOF(data\murica_Multicam.paa);
     };
+		class CLASS(Insignia_naturaltwenty) {
```
Comment (TyroneMF) [inline addons/insignias/CfgUnitInsignia.hpp:396]: Again.

File: addons/insignias/CfgUnitInsignia.hpp [hunk 4]
```diff
@@ -369,6 +393,18 @@ class CfgUnitInsignia {
...
+        MACRO_INSIGNIA_COMMON
+        author = "Lairdd1989";
+        displayName = "Natural Twenty";
+        texture = QPATHTOF(data\naturaltwenty.paa);
+    };	
+		class CLASS(Insignia_neverforget) {
```
Comment (TyroneMF) [inline addons/insignias/CfgUnitInsignia.hpp:402]: And again.

**Review by TyroneMF — CHANGES_REQUESTED** [review]

**Review by TyroneMF — APPROVED** [review]
Looks fine to me.

File: addons/insignias/CfgUnitInsignia.hpp [hunk 5]
```diff
@@ -441,6 +477,12 @@ class CfgUnitInsignia {
         displayName = "Punisher Tactical";
         texture = QPATHTOF(data\punishertactical.paa);
     };
+    class CLASS(Insignia_raptorsquad) {
```
Comment (jonpas) [inline addons/insignias/CfgUnitInsignia.hpp:480]:
```suggestion
    class CLASS(Insignia_RaptorSquad) {
```

**Review by jonpas — CHANGES_REQUESTED** [review]

File: addons/insignias/CfgUnitInsignia.hpp [hunk 6]
```diff
@@ -51,6 +51,18 @@ class CfgUnitInsignia {
         displayName = "Boba Fett (Black-Silver)";
         texture = QPATHTOF(data\bobafett_blacksilver.paa);
     };
+    class CLASS(Insignia_BrandOfSacrifice) {
+        MACRO_INSIGNIA_COMMON
+        author = "Lairdd1989";
+        displayName = "Brand of Sacrifice";
```
Comment (jonpas) [inline addons/insignias/CfgUnitInsignia.hpp:57]:
```suggestion
        displayName = "Brand Of Sacrifice";
```

**Review by jonpas — CHANGES_REQUESTED** [review]

**Review by jonpas — APPROVED** [review]

**Pull request merged**